frontend/.eslintcache

```diff
@@ -1 +1 @@
":"33","messages":"34","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"35","messages":"36","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"37","messages":"38","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"39"},{"filePath":"40","messages":"41","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"42","messages":"43","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"44","messages":"45","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"46","messages":"47","errorCount":1,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"source":null},{"filePath":"48","messages":"49","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"50","messages":"51","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"52","messages":"53","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"54","messages":"55","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"/Users/mcraepv/projects/apprentice-react-o-shop/frontend/src/reportWebVitals.js",[],["56","57"],"/Users/mcraepv/projects/apprentice-react-o-shop/frontend/src/components/Main/Main.js",[],"/Users/mcraepv/projects/apprentice-react-o-shop/frontend/src/components/Header/Header.js",[],"/Users/mcraepv/projects/apprentice-react-o-shop/frontend/src/App.js",[],"/Users/mcraepv/projects/apprentice-react-o-shop/frontend/src/index.js",[],["58","59"],"/Users/mcraepv/projects/apprentice-react-o-shop/frontend/src/pages/ShoppingCart/ShoppingCart.js",[],"/Users/mcraepv/projects/apprentice-react-o-shop/frontend/src/pages/Home/Home.js",[],"/Users/mcraepv/projects/apprentice-react-o-shop/frontend/src/pages/CheckOut/CheckOut.js",[],"/Users/mcraepv/projects/apprentice-react-o-shop/frontend/src/utils/history.js",["60"],"/Users/mcraepv/projects/apprentice-react-o-shop/frontend/src/pages/Products/Products.js",[],"/Users/mcraepv/projects/apprentice-react-o-shop/frontend/src/pages/Orders/Orders.js",[],"/Users/mcraepv/projects/apprentice-react-o-shop/frontend/src/pages/OrderSuccess/OrderSuccess.js",[],"/Users/mcraepv/projects/apprentice-react-o-shop/frontend/src/pages/AdminOrders/AdminOrders.js",[],{"ruleId":"61","replacedBy":"62"},{"ruleId":"63","replacedBy":"64"},{"ruleId":"61","replacedBy":"65"},{"ruleId":"63","replacedBy":"66"},{"ruleId":"67","severity":2,"message":"68","line":3,"column":16,"nodeType":"69","endLine":3,"endColumn":26},"no-native-reassign",["70"],"no-negated-in-lhs",["71"],["70"],["71"],"react-hooks/rules-of-hooks","React Hook \"useHistory\" cannot be called at the top level. React Hooks must be called in a React function component or a custom React Hook function.","Identifier","no-global-assign","no-unsafe-negation"]
```

Comment:
I think this file, and all files, require an extra line at the end of the file.

Reply:
Added extra line and checked other files for extra line as well